spec/factories/uk_studying_completion.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :uk_studying_completion do
```

Comment:
and this one?

Reply:
https://github.com/DFE-Digital/get-teacher-training-adviser-service/blob/master/app/models/uk_studying_completion.rb